apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -59,6 +91,27 @@ class ReviewTab extends Component {
           isReadyForReview: false
         });
       });
+
+    if (
+      !this.props.viewAsCodeReviewer &&
+      this.props.viewAs !== ViewType.Teacher
+    ) {
+      codeReviewDataApi
+        .getReviewablePeers(channelId, serverLevelId, serverScriptId)
+        .done(data => {
+          this.setState({
+            reviewablePeers: _.chain(data)
+              .filter(peerEntry => peerEntry && peerEntry.length === 2)
+              .map(peerEntry => ({id: peerEntry[0], name: peerEntry[1]}))
```

Comment:
Oh no worries, this works too but yeah it may just be cleaner if it's in the controller. I can add a follow up to do that. The `filter` is just there as a precaution so that faulty/missing data doesn't break the client. In theory that check should always pass, but in case it doesn't, our UI wouldn't break.

Reply:
> Also for future reference I think we have optional chaining enabled, so you don't have to check that `peerEntry` exists:
> 
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Ah right good to know!

Reply:
Created https://codedotorg.atlassian.net/browse/CSA-616 to follow up